tests/com/bl/ttt/test_Tictactoe_is_draw.py

```diff
@@ -0,0 +1,17 @@
+from com.bl.ttt.tictactoe import Tictactoe
+
+
+def test_01_is_draw_identifies_draw():
+    g = Tictactoe()
+    g.cells = ['X', 'O', 'X',
+               'X', 'O', 'X',
+               'O', 'X', 'O']
+    assert g.is_draw()
+
+
+def test_02_is_draw_identifies_not_yet_draw():
+    g = Tictactoe()
+    g.cells = ['X', 'O', 'X',
+               'X', 'O', ' ',
+               'O', 'X', 'O']
```

Comment:
What happens when we have a winner and there are still ' ' cells ?

Reply:
test_03_is_draw_identifies_win_not_draw() added